spaces/S000175/properties/P000086.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000175
+property: P000086
+value: true
+---
+
+The group of Euclidean translations acts continuously on $X$ because it preserves lines and open subintervals of lines. The action is clearly transitive.
```

Comment:
```suggestion
Given points $p,q\in\mathbb R^2$, the function $f(x)=x-p+q$ is a bijection mapping $p$ to $q$. Since the function is linear, it preserves lines passing through points and therefore is a homeomorphism.
```